setup.py

```diff
@@ -222,6 +222,8 @@ def package_files(prefix: str, directory: str, extension: str):
     'mlflow>=2.0.1,<3.0',
 ]
 
+extra_deps['pandas'] = ['pandas>=2.0.3']
```

Comment:
do you think we need this minimum pin?

Reply:
Nope! I wasn't too sure about convention, so I started with this, but it seems like it's better to just do equals

Reply:
Oh I actually meant we can probably be a lot less restrictive than this since we are only using pandas very minimally. Like maybe just `pandas<3` is ok? or maybe `>=2` and `<3`?

Reply:
ohhh, gotcha! i'll update it to the latter